man/man1/pkcstok_admin.1.in

```diff
@@ -196,6 +196,50 @@ option to see more detailed messages about the actions the tool is performing.
 .
 .
 .
+.PP
+.SS "Resetting a token to its initial state"
+.B pkcstok_admin
+.B reset
+.BR \-\-token | \-t
+.I TOKNAME
+.RB [ \-\-force ]
+.RB [ \-\-verbose ]
+.PP
+Use the \fBreset\fP command to reset a token to its initial state.
+This also resets all PINs and removes all token objects. Use with care!
+.PP
+Resetting a token can be required if you forgt the SO pin, or the SO pin got
+locked due to too many login attempts using a wrong SO PIN..
```

Comment:
Are the two dots at the end of the sentence intended?

Reply:
no, fixed